hoomd/md/TwoStepNPTMTK.h

```diff
@@ -55,12 +55,12 @@ class PYBIND11_EXPORT TwoStepNPTMTK : public IntegrationMethodTwoStep
          */
         enum baroFlags
             {
-            baro_x = 1,
-            baro_y = 2,
-            baro_z = 4,
-            baro_xy = 8,
-            baro_xz = 16,
-            baro_yz = 32
+            baro_x = 32,
```

Comment:
Why change these?

Reply:
@joaander I thought opposite sequence for baroflags is better to append in the order 'x,y,z,xy,xz,yz' in get function in cc file in line number 717-722. Will there be other built-in function to append in the opposite order?

Reply:
You are right that it makes the order of the bits in flags consistent with the order of bits in the `std::vector<bool>`. There may be some benefit to this, but given the way you explicitly convert the two back and forth the change is not needed.